tekton/pipelines/lh-release-pipeline.yaml

```diff
@@ -0,0 +1,136 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: lh-release-pipeline
+spec:
+  params:
+    - name: version
+      type: string
+      description: The version to release, without a leading "v"
+    - name: previous-version
+      type: string
+      description: The previous version, without a leading "v"
+    - name: revision
+      type: string
+      description: The revision to tag and release
+  workspaces:
+    - name: source
+  tasks:
+    - name: fetch-repo
+      taskRef:
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: source
+      params:
+        - name: url
+          value: "https://github.com/jenkins-x/lighthouse.git"
+        - name: revision
+          value: $(params.revision)
+        - name: subdirectory
+          value: ""
+        - name: depth
+          value: "0"
+    - name: build-binaries
+      taskRef:
+        name: lh-build-binaries
+      runAfter:
+        - fetch-repo
```

Comment:
not really specific to this PR, but is this really required? IMO it should be the default. I'd argue in 99% of the cases I am writing my pipeline in a way that the steps are already in sequence.